<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowEditAnnotation</class>
 <widget class="QWidget" name="WindowEditAnnotation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Annotation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>9</number>
   </property>
   <item row="1" column="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Color Table</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCleanUp">
         <property name="text">
          <string>Clean Up</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeWidgetAvailableLabels">
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QToolBar" name="widgetToolBar">
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <addaction name="actionSave"/>
     <addaction name="separator"/>
     <addaction name="actionUndo"/>
     <addaction name="actionRedo"/>
     <addaction name="separator"/>
     <addaction name="actionPathFill"/>
     <addaction name="actionMakePath"/>
     <addaction name="actionMakeClosedPath"/>
     <addaction name="actionDeletePath"/>
     <addaction name="actionClearMarks"/>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="verticalSpacing">
        <number>4</number>
       </property>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEditName">
         <property name="toolTip">
          <string>Press Return to change</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineEditColor">
         <property name="toolTip">
          <string>Press return to change</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditIndex">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Press return to change</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Index</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QtColorPicker" name="colorpicker" native="true"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButtonSet">
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDelete">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="topMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Annotation</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxShowAll">
         <property name="text">
          <string>All</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeWidgetExistingLabels">
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0,2">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonFromCTab">
       <property name="toolTip">
        <string>Copy To</string>
       </property>
       <property name="text">
        <string>&lt;-</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextOnly</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonLoadSegmentation">
       <property name="text">
        <string>Load From Segmentation</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLoadLUT">
       <property name="text">
        <string>Load Color Table</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionMakePath">
   <property name="icon">
    <iconset resource="freeview.qrc">
     <normaloff>:/resource/icons/surface_path_make.png</normaloff>:/resource/icons/surface_path_make.png</iconset>
   </property>
   <property name="text">
    <string>Make Path</string>
   </property>
  </action>
  <action name="actionMakeClosedPath">
   <property name="icon">
    <iconset resource="freeview.qrc">
     <normaloff>:/resource/icons/surface_path_make_closed.png</normaloff>:/resource/icons/surface_path_make_closed.png</iconset>
   </property>
   <property name="text">
    <string>Make Closed Path</string>
   </property>
  </action>
  <action name="actionDeletePath">
   <property name="icon">
    <iconset resource="freeview.qrc">
     <normaloff>:/resource/icons/surface_path_delete.png</normaloff>:/resource/icons/surface_path_delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete Path</string>
   </property>
  </action>
  <action name="actionClearMarks">
   <property name="icon">
    <iconset resource="freeview.qrc">
     <normaloff>:/resource/icons/surface_clear_marks.png</normaloff>:/resource/icons/surface_clear_marks.png</iconset>
   </property>
   <property name="text">
    <string>Clear Marks</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="freeview.qrc">
     <normaloff>:/resource/icons/edit_undo.png</normaloff>:/resource/icons/edit_undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="freeview.qrc">
     <normaloff>:/resource/icons/edit_redo.png</normaloff>:/resource/icons/edit_redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="freeview.qrc">
     <normaloff>:/resource/icons/annot_save.png</normaloff>:/resource/icons/annot_save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionPathFill">
   <property name="icon">
    <iconset resource="freeview.qrc">
     <normaloff>:/resource/icons/surface_path_fill.png</normaloff>:/resource/icons/surface_path_fill.png</iconset>
   </property>
   <property name="text">
    <string>Custom Fill</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtColorPicker</class>
   <extends>QWidget</extends>
   <header>qtcolorpicker.h</header>
   <container>1</container>
   <slots>
    <signal>colorChanged(QColor)</signal>
    <signal>currentColorChanged(QColor)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditIndex</tabstop>
  <tabstop>lineEditColor</tabstop>
  <tabstop>treeWidgetExistingLabels</tabstop>
  <tabstop>treeWidgetAvailableLabels</tabstop>
  <tabstop>pushButtonSet</tabstop>
  <tabstop>pushButtonDelete</tabstop>
 </tabstops>
 <resources>
  <include location="freeview.qrc"/>
 </resources>
 <connections/>
</ui>
